src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -0,0 +1,179 @@
+authors:
+  - una
+date: 2020-10-14
+hero: hero.jpg
+alt: Set of tools on a desk. Via @yer_a_wizard on Unsplash.
+description: Min, max, and clamp provide some powerful CSS capabilities that enable more responsive styling with fewer liens of code. This post goes over how to control element sizing, maintain proper spacing, and implement fluid typography using these well-supported CSS math functions.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - css
+  - chrome79
+feedback:
+  - api
+---
+
+With responsive design evolving and becoming increasingly nuanced, CSS itself is constantly evolving and providing authors increased control. The [`min()`](https://developer.mozilla.org/en-US/docs/Web/CSS/min), [`max()`](https://developer.mozilla.org/en-US/docs/Web/CSS/max), and [`clamp()`](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp) functions, now supported in all modern browsers, are among the latest tools in making authoring websites and apps more dynamic and responsive. 
+
+When it comes to flexible and fluid typography, controlled element resizing, and maintaining proper spacing, `min()`, `max()`, and `clamp()` can help.
+
+## Background
+
+The math functions, `calc()`, `min()`, `max()`, and `clamp()` allow mathematical expressions with addition (+), subtraction (-), multiplication (*), and division (/) to be used as component values — [[CSS Values And Units Level 4]](https://www.w3.org/TR/css-values-4/#calc-notation). 
+
+Safari was the first to [ship](https://bugs.webkit.org/show_bug.cgi?id=167000) the complete set of functions in April 2019, with Chromium following later that year in version 79. This year, with Firefox [75](https://bugzilla.mozilla.org/show_bug.cgi?id=1519519) shipping, we now have browser parity for `min()`, `max()`, and `clamp()` in all evergreen browsers.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./caniuse.png" alt="" />
+  <figcaption class="w-figcaption">
+    <a href="https://caniuse.com/?search=min%20max%20clamp">Caniuse</a> support table.
+  </figcaption>
+</figure>
+
+## Usage
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-demo.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Showing how the min() function selects a value based on a list of options and its parent. <a href="https://codepen.io/una/pen/rNeGNVL">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+You can use `min()`, `max()`, and `clamp()` on the right hand side of any CSS expression where it would make sense. For `min()` and `max()`, you provide an argument list of values, and the browser determines which one is either the smallest or largest, respectively. For example, in the case of: `min(1rem, 50%, 10vw)`, the browser would calculate which of these relative units is the smallest, and use that value as the actual value.
```

Comment:
```suggestion
You can use `min()`, `max()`, and `clamp()` on the right hand side of any CSS expression where it would make sense. For `min()` and `max()`, you provide an argument list of values, and the browser determines which one is either the smallest or largest, respectively. For example, in the case of: `min(1rem, 50%, 10vw)`, the browser calculates which of these relative units is the smallest, and uses that value as the actual value.
```